shardingsphere-sql-parser/shardingsphere-sql-parser-dialect/shardingsphere-sql-parser-oracle/src/main/antlr4/imports/oracle/DDLStatement.g4

```diff
@@ -919,3 +919,31 @@ flashbackArchiveClause
 alterSynonym
     : ALTER PUBLIC? SYNONYM (schemaName DOT_)? synonymName (COMPILE | EDITIONABLE | NONEDITIONABLE)
     ;
+
+alterTablePartition
+    : addTablePartition | dropTablePartition
+    ;
+
+addTablePartition
+    : ADD (addRangePartitionClause | addListPartitionClause)
+    ;
+
+addRangePartitionClause
+    : PARTITION partitionName? rangeValuesClause tablePartitionDescription
+    ((LP_? rangeSubpartitionDesc (COMMA_ rangeSubpartitionDesc)* | listSubpartitionDesc (COMMA_ listSubpartitionDesc)* | individualHashSubparts (COMMA_ individualHashSubparts)* RP_?)
+        | hashSubpartitionQuantity)?
+    ;
+
+addListPartitionClause
+    : PARTITION partitionName? listValuesClause tablePartitionDescription
+    ((LP_? rangeSubpartitionDesc (COMMA_ rangeSubpartitionDesc)* | listSubpartitionDesc (COMMA_ listSubpartitionDesc)* | individualHashSubparts (COMMA_ individualHashSubparts)* RP_?)
+    | hashSubpartitionQuantity)?
+    ;
+
```

Comment:
The hash partition and system partition also need to be added.

Reply:
The addRangePartitionClause and addListPartitionClause are not fully completed, below rules need to be done.
table_partition_description
external_part_subpart_data_props
update_index_clauses

Reply:
@wgy8283335 
I know, but I just used the existing syntax to build it. If want to support fully completed grammer maybe need @tristaZero open a issues.
By the way, the grammer of partition in `CREATE TABLE` are also not support  fully completed grammer.

Reply:
Hi, We can have a plan to support this grammar completely. Firstly, we can add the whole grammar definition, then we can consider improve the class of `CreateTableStatement`.

Reply:
@tristaZero agree, this PR can merge first?

Reply:
So 
```
The addRangePartitionClause and addListPartitionClause are not fully completed, below rules need to be done.
table_partition_description
external_part_subpart_data_props
update_index_clauses
```
Will these definitions be improved in the next PR?
It is not a complete grammar definition, is it?

Reply:
Next PR complete the grammer of partition in `CreateTableStatement` and `AlterTableStatement` .

Reply:
Great.